api/graphql/makeModels.js

```diff
@@ -125,6 +139,21 @@ export default function makeModels (userId, isAdmin) {
         url: c => c.get('url'),
         imageUrl: c => c.get('image_url')
       }
+    },
+
+    MessageThread: {
+      model: Post,
+      attributes: ['id', 'created_at', 'updated_at'],
+      relations: [
+        'followers',
```

Comment:
We could leave this as 'followers' but in my front end work I was inclining towards calling them 'participants' which lends a little more clarity in a 'messaging' context. This seems like an opportunity to do that here too if we wanted...